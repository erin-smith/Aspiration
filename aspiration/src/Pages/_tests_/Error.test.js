import * as React from "react";
import ErrorPage from "../error";
import renderer from "react-test-renderer";

describe("Error page", () => {
  test("Should match the snapshot", () => {
    const tree = renderer.create(<ErrorPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
