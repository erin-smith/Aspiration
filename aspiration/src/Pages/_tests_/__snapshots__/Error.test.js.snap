// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page Should match the snapshot 1`] = `
<div>
  <h1>
    404 - Not Found!
  </h1>
</div>
`;
